zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/CamundaExporter.java

```diff
@@ -53,6 +55,15 @@ public class CamundaExporter implements Exporter {
   private ExporterBatchWriter writer;
   private long lastPosition = -1;
   private final ExporterResourceProvider provider;
+  private CamundaExporterMetrics metrics;
+
+  /**
+   * Sample to measure the flush latency of the current bulk request.
+   *
+   * <p>Time of how long an export bulk request is open and collects new records before flushing,
+   * meaning latency until the next flush is done.
+   */
+  private Sample flushLatencyMeasurement;
```

Comment:
💭   Could we keep track of this instance inside the `CamundaExporterMetrics `?

Reply:
Yes if we do not ever want to track more than 1 instance of flush latency at a time, do you think this will ever occur?

Reply:
I'm not sure I completely understand your point. so let me rephrase

We have one instance of `CamundaExporterMetrics` per instance of `exporter`, right? and basically `flushLatencyMeasurement` is necessarily one per exporter instance too. How is having here different than there? are we sharing this `exporter metrics` instance somehow?

Please elaborate to me if there's something I might be missing 🙏

Reply:
I was thinking of a scenario where we would have multiple batch requests (per exporter) at once  (for whatever reason), and we would want to track the latencies of each. But this probably won't happen so I'll move it to metrics